### PR TITLE
Add pretext graph tool

### PR DESCRIPTION
**Pull request by astrovsky01**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [x] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/pretext/pretext_graph.xml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+            <test expect_num_outputs="1">
+                <param name="input" value="input_graph.bgr" ftype="bedgraph"/>
+                <param name="pretext" value="input_graph.pretext" ftype="pretext"/>
+                <param name="name" value="testing_2"/>
+                <output name="graph_out" file="output_5.pretext"/>
+            </test>
+    </tests>
+    <help><![CDATA[
+        Adds additional data onto existing pretextmap files.
+        -i input Pretext file, required. Sequence names in the Pretext file must match sequence names in the bedgraph data; although relative sort order is unimportant.
```
Comment (bgruening) [inline tools/pretext/pretext_graph.xml:50]: This help does not make much sense in the Galaxy context. Can you adjust that and offer some more useful informations for the user?

**Comment by astrovsky01** [comment]
Feel like I'm crazy here -- it's passing locally and I don't know why it can't find the files

**Comment by astrovsky01** [comment]
I have been, yes, and they've been passing. Switching to cp also passes locally, so let's see how that does here

File: tools/pretext/pretext_graph.xml [hunk 2]
```diff
@@ -9,9 +9,9 @@
         <requirement type="package" version="377"> ucsc-bigwigtobedgraph</requirement>
     </requirements>
     <command detect_errors="exit_code"><![CDATA[
-        ln -s '$pretext' input.pretext &&
+        cp '$pretext' input.pretext &&
```
Comment (bgruening) [inline tools/pretext/pretext_graph.xml:12]: I think we can switch back?

Comment (astrovsky01) [reply tools/pretext/pretext_graph.xml]: Looks like we can't for some reason

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Thanks @astrovsky01

**Pull request merged**